blocks/tabs/tabs.js

```diff
@@ -5,6 +5,20 @@ function hasWrapper(el) {
   return !!el.firstElementChild && window.getComputedStyle(el.firstElementChild).display === 'block';
 }
 
+function embedWistia(url) {
```

Comment:
can we have one function for the embedding part
for columns, video and the tabs block ?